packages/slice-machine/components/Simulator/components/Toolbar/ScreensizeInput/index.tsx

```diff
@@ -0,0 +1,88 @@
+import { ScreenDimensions } from "@lib/models/common/Screenshots";
+import { Flex, Input, Label, ThemeUIStyleObject } from "theme-ui";
+
+export enum ScreenSizeOptions {
+  DESKTOP = "Desktop",
+  TABLET = "Tablet",
+  MOBILE = "Mobile",
+  CUSTOM = "Custom",
+}
+
+export const ScreenSizes: Record<string, ScreenDimensions> = {
+  [ScreenSizeOptions.DESKTOP]: {
+    width: 1280,
+    height: 800,
+  },
+  [ScreenSizeOptions.TABLET]: {
+    width: 1080,
+    height: 810,
+  },
+  [ScreenSizeOptions.MOBILE]: {
+    width: 390,
+    height: 844,
+  },
+};
+
+type ScreensizeInputProps = {
+  label: "H" | "W";
+  startValue: number;
+  onChange: (sizeEvent: React.ChangeEvent<HTMLInputElement>) => void;
+  sx?: ThemeUIStyleObject;
+};
+
+export const ScreensizeInput: React.FC<ScreensizeInputProps> = ({
+  label,
+  startValue,
+  onChange,
+  sx,
+}) => {
+  return (
+    <Flex
+      sx={{
+        alignItems: "center",
+        ...sx,
+      }}
+    >
+      <Label
+        sx={{
+          backgroundColor: "grey07",
+          border: "transparent",
+          borderRight: "1px solid #DCDBDD",
```

Comment:
We have to avoid hardcoded colors as much as possible to use the maximal performances of the theme.

```suggestion
          borderRight: (t) => `1px solide ${String(t.colors?.myvariable)}`,
```